src/utils/json-big.ts

```diff
@@ -0,0 +1,62 @@
+import JsonBig from '@aeternity/json-bigint'
+import BigNumber from 'bignumber.js'
+
+const jsonBig = JsonBig({ storeAsString: true })
+
+interface BigNumValueObject {
+  jsonrpc: string
+  method: string
+  params?: {
+    action?: string
+    signed_tx?: string
+    code?: string
+    call_data?: string
+    deposit?: BigNumber
+    vm_version?: number
+    abi_version?: number
+    amount?: number
+    gas_price?: number
+    gas?: number
+    error?: number
+    caller_id?: string
+    contract_id?: string
+    round?: number
+    pubkey?: string
+    data?: {
+      channel_id?: string
+      error_code?: number
+      error_msg?: string
+      round?: number
+    }
+  }
+  id?: string
+  version?: number
+}
```

Comment:
JSON converter shouldn't be aware of aeternity-specific types, I don't think this is needed